daemon/daemon.go

```diff
@@ -549,20 +549,11 @@ func (daemon *Daemon) GetByName(name string) (*container.Container, error) {
 
 // getEventFilter returns a filters.Filter for a set of filters
 func (daemon *Daemon) getEventFilter(filter filters.Args) *events.Filter {
-	// incoming container filter can be name, id or partial id, convert to
-	// a full container id
-	for _, cn := range filter.Get("container") {
-		c, err := daemon.GetContainer(cn)
-		filter.Del("container", cn)
-		if err == nil {
-			filter.Add("container", c.ID)
-		}
-	}
-	return events.NewFilter(filter, daemon.GetLabels)
```

Comment:
I think this method `getEventFilter()` and `daemon.GetLabels()` can both be delete now.